Slim/Plugin/MusicMagic/Plugin.pm

```diff
@@ -784,8 +790,8 @@ sub mixExitHandler {
 
 sub getMix {
 	my $client = shift;
-	my $id = shift;
 	my $for = shift;
+	my @ids = splice(@_);
 
```

Comment:
Here you'd not change the order of the parameters, but check what `$id` was, something like this:

```perl
my @ids = ref $id ? @$id : ($id);
```